space-framework/docs/space-framework.md

```diff
@@ -0,0 +1,92 @@
+2. Imported spaces: A Space can be created by importing an existing pSpace. In this case the Space is not linked to any space provider and the client who imports that Space need to supply the access information that allows other clients to connect to the pSpace. For imported spaces the desired state is derived by the pSpace. This means that the SF is not responsible for the life cycle management of the pSpace but only update the corresponding Space object. For example, when deleteing the space object of an existing imported pSpace the space framework will recreate the space object. 
+3. Unmanaged Space: The SF supports discovery of existing pSpaces on a space provider and adding them as "Unmanaged Spaces" into the framework. This mode is similar to the import mode in terms of the "source of truth" - the desired state is defined by the pSpace. However, as oppose to import, here the pSpaces are automatically discovered through the space provider regular interface and the pSpace access information is retrieved from teh space provider similary to how it is done in the managed space case. The SF supports pattern based filtering rules to limit the discovery of pSpaces. 
+
+### Space Manager 
+The Space Manager (SM) is a Kuberentes controller that is responsible for maintaing the state of the Space objects. The SM reconcile the Space and SpaceProviderDesc objects, and monitor the state of the pSpaces through the space provider.   
+The SM uses a library of space provider adaptors to communicate with the space providers. Currently the SF includes 3 space provider adaptors KCP, KIND, and KubeFlex.
+
+### Space provider adaptors
+The space manager uses a set of space provider adaptors that interacts with the space provider. All space provider adaptors implement a simple "space client interface" that includes basic pSpace life cycle operatios. In addition the interface also defines the events that sent from the adaptors to the space manager.
+
+### Space aware client (SAC)
+The Space aware client (msclient) allows clients/controllers to easily get access to the underlying space by simply using the space name. There is no direct interaction between the msclient and the Space  provider, and therefore the msclient is transparent to the specific provider of the spaces. 
+![Space Aware Client](SAC.drawio.svg "Space Aware Client")
+The only requirement is that the space can be accessed through regular Kube APIs when using the appropriate kubeconfig information. 
+
+**Main features**  
```

Comment:
Why not use markdown markup on this subsection header?

Reply:
Please do not mark a conversation as "resolved" without a clue about _why_ it is being marked as resolved.

Reply:
I agree. I tend to resolve only when I changed/fix the exact proposal in the comments (mainly spelling mistakes). If that was not the case this is a mistake

Reply:
Regarding the markdown comment - I wasnt sure what you meant here

Reply:
I mean that what I see here currently is `**Main features**` and I think this intended to be the heading of a subsection nested under `### Space aware client (SAC)`. So my question is why not continue with regular markdown for section headings and use `#### Main features`?

Reply:
Oh I see. Changed to markdown